BULK VALIDATION description-missing (#1430)
dotnet/docs-desktop: dotnet-desktop-guide/framework/winforms/controls/checkedlistbox-control-windows-forms.md

---
title: "CheckedListBox Control"

> description: Learn more about the CheckedListBox control in Windows Forms, which is similar to the ListBox control.

Reviewer: ```suggestion
description: Learn about which articles will help you understand more about the CheckedListBox control in Windows Forms.
```
